server/controllers/list.js

```diff
@@ -1210,49 +1216,15 @@ const getArchivedItems = (req, resp) => {
       }
 
       const { items } = list;
-      const archivedItems = items.filter(item => item.isArchived);
+      const archivedItems = items.filter(
```

Comment:
Please split filter to `liveItems = items.filter(({ isDeleted }) => !isDeleted)`, then `archivedItems = liveItems.filter(({ isArchived }) => isArchived)`